frontend/.gitignore

```diff
@@ -28,3 +28,6 @@ coverage
 
 # vscode
 .vscode
+
+# tailwindcss
+src/tailwind.output.css
```

Comment:
Can we put it into a build or generated folder and ignore the folder instead?

Reply:
Sounds good, created build folder to put this file to there.